.asciidoctor/diagram/diag-6634a60eba787f2c6f5ca38a56944b14.png.cache

```diff
@@ -0,0 +1 @@
+{"checksum":"6634a60eba787f2c6f5ca38a56944b14","options":{"scale":null,"tabs":null,"background":null,"antialias":null,"separation":null,"round_corners":null,"shadows":null,"debug":null,"fixed_slope":null,"transparent":null},"width":550,"height":280}
```

Comment:
the .asciidoctor/ folder can probably be removed, since it'll be regenerated during the build that's triggered by GitHub actions

Reply:
Best solution might be to just track .asciidoctor/ in .gitignore!